### ISSUE TITLE
超参数设置问题

### ISSUE THREAD

**Issue by xq245** [kwuking/TimeMixer#192]
请问每个实验最佳的超参数设置是什么，为什么我好像一直复现不出最优的效果

**Comment by kevinliu2000** [comment]
Thanks for your interest. I think there might be two factors affecting the these. One is that the current PyTorch version and CUDA environment could influence the outcome. Another is that the learning rate in the script seems quite large, which may cause non-convergence on certain devices. You could try using a smaller learning rate, such as 0.001.
